plugins/tech-insights-backend-module-jsonfc/src/service/JsonRulesEngineFactChecker.ts

```diff
@@ -117,7 +117,7 @@ export class JsonRulesEngineFactChecker
       if (hasAllFacts) {
         engine.addRule({ ...techInsightCheck.rule, event: noopEvent });
       } else {
-        this.logger.warn(
+        this.logger.debug(
```

Comment:
My reasoning here as I know this will be asked:

Fact Retrievers can target a subset entities by using filters, so it's expected that not all facts to be available on all entities, so it's ok for checks to be skipped.

When I see a warn on logs, It's something I want to dig into and fix it, but in this scenario, there is no fix and the result is logs being spammed with unactionable logs.

Reply:
Sounds reasonable :)

Reply:
by the way, I tested this operation with our setup of 200+ entities and 9 checks, it takes 1.2sec with debug log and 3.5sec when using warn